LearnositySDKUnitTests/RequestAsStringUnitTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using LearnositySDK.Utils;
+using LearnositySDK.Request;
+
+
+namespace LearnositySDKUnitTests
+{
+    [TestClass]
+    public class RequestAsStringUnitTests
+    {
+        private readonly string consumerKey = "yis0TYCu7U9V4o7M";
+        private readonly string consumerSecret = "74c5fd430cf1242a527f6223aebd42d30464be22";
+        private readonly string domain = "localhost";
+
+        [TestMethod]
+        public void AuthorApiItemListAsString()
+        {
+            // Arrange
+            string service = "author";
+            JsonObject security = GetSecurityObj();
+            string secret = consumerSecret;
+            JsonObject request = new JsonObject();
```

Comment:
Could you separate the building of the test requests from the main test?

Ideally, we'd want something like data providers, where the separate method would return something like `request` and `expectedOutput` in one step, after which you can do something like this (pseudocode)

```
request, expectedOutput = AuthorApiItemListAsStringProvider()

Init initAsString = new Init(service, security.toJson(), secret, request.toJson());
signedRequestAsString = initAsString.generate();

Assert.AreEqual(
                signedRequestAsString,
                expectedOutput
            );
```

Also, it might just be easier to have `AuthorApiItemListAsStringProvider` output a JSON string, rather than building the object manually, then serialising it.